src/stable/test/security_checks.ts

```diff
@@ -0,0 +1,18 @@
+import { execSyncPretty } from '#utils/exec_sync_pretty';
```

Comment:
Why are we only running npm audit? Shouldn't we run the cargo checks locally as well?...oh wait...well let's just talk about it.

Reply:
if you wanted to run it locally I would recommend running it as part of the template creation

Reply:
That makes sense, let's do that